plugin/hooks.go

```diff
@@ -133,6 +134,11 @@ type Hooks interface {
 	// UserHasLoggedIn is invoked after a user has logged in.
 	UserHasLoggedIn(c *Context, user *model.User)
 
+	// UserWasCreated is invoked after a user was created.
+	//
+	// Minimum server version: 5.10
```

Comment:
If we get this merged until the 30th, it will go into 5.8.

Reply:
Done. I hope it will be merged until the 30th :).

Reply:
Code complete hast been moved to this Monday. Because of this, this feature will so into 5.10. Sorry for making your false hope.